packages/mint-components/src/components/sqm-share-code/sqm-share-code.tsx

```diff
@@ -45,11 +42,47 @@ export class ShareCode {
     attribute: "tooltip-lifespan",
   })
   tooltiplifespan: number = 1000;
+  /**
+   * Change the text alignment
+   *
+   * @uiName Align text
+   * @uiType string
+   * @uiEnum ["left", "center"]
+   * @uiEnumNames ["left", "center"]
```

Comment:
uiEnumNames Should be title case (make sure that's the case for the other two components with this prop too)